### PR TITLE
l10n: zh-TW localization update

### PR DESCRIPTION
**Pull request by visig9**

### CODE REVIEW

File: src/partials/language/zh-TW.html [hunk 1]
```diff
@@ -31,13 +32,16 @@
   "meta.comments": "留言",
   "meta.source": "來源",
   "search.language": "ja",
+  "search.pipeline.stopwords": false,
```
Comment (squidfunk) [inline src/partials/language/zh-TW.html]: Could you explain why you added those?

File: src/partials/language/zh-TW.html [hunk 2]
```diff
@@ -23,6 +23,7 @@
 <!-- Translations: Chinese (Taiwanese) -->
 {% macro t(key) %}{{ {
   "language": "zh-Hant",
+  "direction": "ltr",
```
Comment (squidfunk) [inline src/partials/language/zh-TW.html]: This can be removed, `ltr` is set by default.

File: src/partials/language/zh-TW.html (hunk 1 above)
Comment (visig9) [reply src/partials/language/zh-TW.html]: Nice catch! It my fault. I re-check the code base and this line should be deleted in chinese context, and also `trimmer` line.

Long story short, I found `zh-TW.html` missing the `source.revision.date` and cause `git-revision-date-localized` plugin not work correctly (no default string in `5.0.0rc3`). So I think should not leave any fields missing in this file and try to copy some line from other file. And when I test, the search result still not too bad. :P

Anyway this should be clear out. Thanks so much. I will update this PR ASAP.

**Comment by squidfunk** [comment]
LGTM!

**Pull request merged**